packages/docs/celo-holder-guide/voting-validators.md

```diff
@@ -1,22 +1,60 @@
-# Voting on Validator Groups
-
-## Proof of Stake Election Essentials
+# Voting for Validator Groups
```

Comment:
Note this is ignored... it takes the title from the SUMMARY.md file. if you change it there, you'll need to also add redirects.